### DISCUSSION TITLE
dag task end, but not close timely

### DISCUSSION THREAD

**Discussion by huang1125677925** [apache/airflow#22235] [General]
### Apache Airflow version

2.2.3

### What happened

i have some day in my linux computer, ervery dag will run one time per 1 minute, but some minutes late，every day will have many tasks maintain running state , not close, I don't know what happend,  if some friends know, please tell me what I should to do?

### What you expected to happen

I hope my dag can run fluently, but I  can't control it, it is not good

### How to reproduce

you can start some high loader dag, and observe it

### Operating System

CentOS release 6.3 (Final)

### Versions of Apache Airflow Providers

2.2.3

### Deployment

Other

### Deployment details

I deploy on an host 

### Anything else

no

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Not reproducible and not clear. This is at most a discussion, but you will need to provide more information if you want to get someone's halp. Logs, deployment details etc.
